plugins/livelog/livelog.go

```diff
@@ -90,6 +90,14 @@ func (tp *taskPlugin) Prepare(context *runtime.TaskContext) error {
 }
 
 func (tp *taskPlugin) Finished(success bool) error {
+	return tp.uploadLog(success)
+}
+
+func (tp *taskPlugin) Exception(runtime.ExceptionReason) error {
+	return tp.uploadLog(false)
```

Comment:
Nice, but let's add a boolean to ensure we don't upload the logs twice, that'll likely cause the queue to return an error for no good reason...

Actually, I'm not sure that it's not a bug, if `Finished` doesn't get called. (we might fix that later)